sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Common/EventTriggerMetrics.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using System.Runtime.Versioning;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents
+{
+    /// <summary>
+    /// Static class to set the metric headers for each event trigger.
+    /// </summary>
+    public static class EventTriggerMetrics
+    {
+        /// <summary>
+        /// The client library's product name
+        /// </summary>
+        public static string ProductName = "AuthenticationEvents";
+
+        /// <summary>
+        /// Current executing assembly
+        /// </summary>
+        private static readonly Assembly assembly = typeof(EventTriggerMetrics).Assembly;
+
+        /// <summary>
+        /// Get the platform of the event trigger based on the OS.
+        /// </summary>
+        /// <returns>OS Name</returns>
+        private static string Platform => RuntimeInformation.OSDescription ?? "unknown";
+
+        /// <summary>
+        /// Product version of the event trigger. Example: 1.0.0-beta, 1.0.0, 2.0.0
+        /// </summary>
+        private static string ProductVersion => assembly.GetCustomAttribute<AssemblyFileVersionAttribute>()?.Version;
+
+        /// <summary>
+        /// Runtime of the event trigger. Example: .NET, JS, TS, PY
+        /// </summary>
+        private static string Framework => assembly.GetCustomAttribute<TargetFrameworkAttribute>()?.FrameworkName;
```

Comment:
This is not a runtime of event trigger. Please either change the description or implementation